templates/Dockerfile-run

```diff
@@ -1,5 +1,18 @@
-## opencore-amr https://sourceforge.net/projects/opencore-amr/
-RUN \
+ENV SRC=/usr/local
+
+### nasm install
+ARG NASM_VERSION=2.14.02
+RUN set -xe && \
```

Comment:
the -e is not necessary since we're using `&&` and the `-x` adds a lot of noise

Reply:
`&&` is not always present `-e` gives guarantees.
`-x` gives an understanding at which step the assembly fell. 
And by comparison with compilation noise, `-x` is practically silent. 
I think this can be left, but if you are against, I will delete.